crates/solidity/inputs/schema/grammar/02-definitions/08-functions/productions.yml

```
@@ -118,6 +118,37 @@
       close:
         reference: "CloseBrace"
 
+- name: "FunctionCallOptions"
+  kind: "Parser"
+  versioned:
+    0.4.11:
+      oneOrMore:
+        delimitedBy:
+          open:
+            reference: "OpenBrace"
+          expression:
+            optional:
+              separatedBy:
+                separator:
+                  reference: "Comma"
+                expression:
+                  reference: "NamedArgument"
+          close:
+            reference: "CloseBrace"
+    0.8.0:
+      delimitedBy:
+        open:
+          reference: "OpenBrace"
+        expression:
+          optional:
+            separatedBy:
+              separator:
+                reference: "Comma"
+              expression:
+                reference: "NamedArgument"
+        close:
+          reference: "CloseBrace"
```

I wonder if we can just reuse `NamedArgumentList` here?


```suggestion
- name: "FunctionCallOptions"
  kind: "Parser"
  versioned:
    0.4.11:
      oneOrMore:
        delimitedBy:
          open:
            reference: "OpenBrace"
          expression:
            optional:
              separatedBy:
                separator:
                  reference: "Comma"
                expression:
                  reference: "NamedArgument"
          close:
            reference: "CloseBrace"
    0.8.0:
      delimitedBy:
        open:
          reference: "OpenBrace"
        expression:
          optional:
            separatedBy:
              separator:
                reference: "Comma"
              expression:
                reference: "NamedArgument"
        close:
          reference: "CloseBrace"
```

---

I didn't want the second node wrapping in the CST